app/controllers/avo/actions_controller.rb

```diff
@@ -4,9 +4,15 @@ module Avo
   class ActionsController < ApplicationController
     before_action :set_resource_name
     before_action :set_resource
+    before_action :set_model, only: :show, if: ->(request) do
```

Comment:
[Link to commit discussion](https://github.com/avo-hq/avo/commit/eccb22901a7e3cd369cb46b4b25c6d5793b447d5#r83390129)